website_form_builder/migrations/1.2.0/post-migrate.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2020 Tecnativa - Jairo Llopis
```

Comment:
Incorrect migration folder!!! It should be fully qualified, or if not, we can have unpredictable migration

Reply:
I did this on purpose, to be able to cherry-pick this to v12, which is also affected.

Reply:
Please do it the other way. Always fully qualified. When cherry-picking, you can do `git mv`. You don't even know if which version v12 is (the numbering is independent).

Reply:
Done.